tests/todo.test.js

```diff
@@ -1,78 +1,61 @@
-const request = require('supertest');
-const server = require('../server');
+const supertest = require('supertest');
+const app = require('../server');
+const request = supertest(app);
+const mongoose = require('mongoose');
+const Todos = require('../models/todo');
 
+let realDbData;
+
+beforeAll(async () => {
+    const url = `mongodb+srv://${process.env.MONGO_LOGIN}:${process.env.MONGO_PASSWORD}@${process.env.CLUSTER_NAME}/${process.env.DB_NAME}`;
+    await mongoose.connect(url, { useUnifiedTopology: true, useNewUrlParser: true, useFindAndModify: true });
+    realDbData = await request.get('/todos');
+    await mongoose.disconnect();
+    const testUrl = `mongodb+srv://${process.env.MONGO_LOGIN}:${process.env.MONGO_PASSWORD}@${process.env.CLUSTER_NAME}/tests`;
+    await mongoose.connect(testUrl, { useUnifiedTopology: true, useNewUrlParser: true, useFindAndModify: true });
+});
+beforeEach(async () => {
+    await Todos.deleteMany();
+    await Todos.insertMany(realDbData.body);
```

Comment:
Why are you copy real DB data? When your real DB changes, tests will failed. You have to insert constant data into your test database, not dynamic data